Letterbook.Adapter.Db/RelationalContext.cs

```diff
@@ -32,29 +31,19 @@ public class RelationalContext : DbContext
     // Called by the designer to create and run migrations
     internal RelationalContext(DbContextOptions<RelationalContext> context) : base(context)
     {
-        _config = new DesignDbOptions();
     }
 
     // Called by DI for normal use
-    public RelationalContext(IOptions<DbOptions> config)
+    public RelationalContext()
     {
-        _config = config.Value;
-    }
-
-
-    protected override void OnConfiguring(DbContextOptionsBuilder options)
-    {
-        var builder = new NpgsqlDataSourceBuilder(_config.GetConnectionString());
-        builder.EnableDynamicJson();
-        options.UseNpgsql(builder.Build());
```

Comment:
This is the really critical bit. We no longer inject `IOptions<DbOptions>` to the `RelationalContext`, which winds up having significant performance impact under heavy load